eng/Versions.props

```diff
@@ -207,6 +207,16 @@
     <!-- ILLink -->
     <MicrosoftNETILLinkTasksVersion>7.0.100-1.22518.2</MicrosoftNETILLinkTasksVersion>
     <MicrosoftNETILLinkAnalyzerPackageVersion>$(MicrosoftNETILLinkTasksVersion)</MicrosoftNETILLinkAnalyzerPackageVersion>
+    <!-- Mono Cecil -->
+    <MicrosoftDotNetCecilVersion>0.11.4-alpha.22524.1</MicrosoftDotNetCecilVersion>
+    <MicrosoftDotNetCecilPdbVersion>0.11.4-alpha.22524.1</MicrosoftDotNetCecilPdbVersion>
+    <!-- ILLink dependencies -->
+    <MicrosoftBuildFrameworkVersion>17.0.0-preview-21267-01</MicrosoftBuildFrameworkVersion>
+    <MicrosoftBuildUtilitiesCoreVersion>17.0.0-preview-21267-01</MicrosoftBuildUtilitiesCoreVersion>
+    <MicrosoftDotNetApiCompatVersion>8.0.0-beta.22473.1</MicrosoftDotNetApiCompatVersion>
+    <MicrosoftCodeAnalysisCSharpAnalyzerTestingXunitVersion>1.0.1-beta1.*</MicrosoftCodeAnalysisCSharpAnalyzerTestingXunitVersion>
```

Comment:
We don't use floating versions in runtime. Please use an exact one. If we want to keep these updated outside of dependency flow, we can leverage Dependabot.

Reply:
The current version is from 2021 so I figure out that it's not important to change it often, I end up using the current exact version.